test/controllers/TaxCodeChangeControllerSpec.scala

```diff
@@ -152,9 +152,7 @@ class TaxCodeChangeControllerSpec extends BaseSpec with ControllerViewTestHelper
         mcc,
         taxCodeComparisonView,
         yourTaxFreeAmountView,
-        whatHappensNextView,
-        partialRetriever,
-        templateRenderer
+        whatHappensNextView
```

Comment:
don't we still need `templateRenderer`?

Reply:
Moved templateRenderer to implicit block